content/2_Bronze/Intro_DS.mdx

```diff
@@ -159,8 +159,80 @@ dynamic arrays (e.g. `vector<vector<int>>`) static arrays of dynamic arrays
 (e.g. `array<vector<int>,5>`), dynamic arrays of static arrays (e.g.
 `vector<array<int,5>>`), and so on.
 
-<!-- Usually, the choice between a static array and a dynamic array is just personal preference. -->
+## Doubly-Linked Lists
+	
+An alternative for variable-length arrays are "doubly-linked lists." They are implemented by "linking" or pointing each element to the element before and after it. We recommend you look at [this Wikipedia article](https://en.wikipedia.org/wiki/Doubly_linked_list) for more information. These support $\mathcal{O}(1)$ insertion and removal if you have access to the pointer of the element beforehand.
+	
+<Warning>
+	Doubly-Linked lists cannot be accessed by indexes. This is a major reason why they are not used, as opposed to vectors or regular arrays.
+</Warning>
+	
+The C++ object for a doubly-linked list is the `list` object.
+
+```cpp
+list<int> li
+```
+
+Alternatively, you could implement such a list on your own:
```

Comment:
I don't think pointers are ever mentioned before this (aside from the note that they are not the same as iterators).

Reply:
I was asked to show an implementation with pointers by @ar492

Reply:
but readers don't necessarily know what a pointer is at this point? at least link to https://www.learncpp.com/cpp-tutorial/introduction-to-pointers/ or something

Reply:
Hm. Ask @ar492